docs/onnx_model_zoo.md

```diff
@@ -0,0 +1,41 @@
+## KotlinDL ONNX Model Zoo
+
+| Model                     |     Android support      |
+|---------------------------|:------------------------:|
+| EfficientNet4Lite         |    :heavy_check_mark:    |
```

Comment:
Perhaps, we could group models by task (classification, object detection, pose detection, etc)?